src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html

```diff
@@ -40,20 +40,8 @@
 
         {% block course_glimpse_footer %}
         <div class="course-{{ course_variant }}-footer">
-            <div class="course-{{ course_variant }}-footer__date">
-                {{ course_state.text|capfirst }}
-                <span>{{ course_state.datetime|date:"DATE_FORMAT" }}</span>
-            </div>
-            {% if course_state.call_to_action %}
-            <div class="course-{{ course_variant }}-footer__cta">
-                <span class="button">
-                    {{ course_state.call_to_action|capfirst }}
-                    <svg aria-hidden="true" role="img" class="icon">
-                        <use xlink:href="{% static 'richie/icons.svg' %}#icon-arrow-right" />
-                    </svg>
-                </span>
-            </div>
-            {% endif %}
+            {{ course_state.text|capfirst }}
+            {{ course_state.datetime|date:"DATE_FORMAT" }}
```

Comment:
Why do you remove the call_to_action ? I thought it was required for some project like funmooc and also the demo (which should demonstrate its capacities). Maybe in future, enable/disable it from a setting ?

Reply:
The footer was sometimes 1 line, sometimes 2... I think that people are aware in 2020 that they should click on a glimpse to see the detail... So I solved the first problem by removing this useless cta.